art/estimators/classification/pytorch.py

```diff
@@ -442,6 +462,24 @@ def get_activations(self, x, layer, batch_size=128):
 
         return results
 
+    def custom_gradient(self, nn_function, tensors, input_values):
+        """
+        Returns the gradient of the nn_function with respect to model input
+
+        :param nn_function: an intermediate tensor representation of the function to differentiate
+        :type nn_function: a Pytorch tensor
+        :param tensors: the tensors or variables to differentiate with respect to
+        :type tensors: `list`
+        :param input_values: the inputs to evaluate the gradient
+        :type input_values: `list`
+        :return: the gradient of the function w.r.t vars
+        :rtype: `np.ndarray`
+        """
+        raise NotImplementedError
+
+    def get_input_layer(self):
+        return self._input
+
```

Comment:
Please remove these lines because `custom_gradient` is so far only part of `KerasClassifier` and not the Estimator API.
```suggestion
```